src/components/LoginForm/PersonalFormFields.tsx

```diff
@@ -0,0 +1,120 @@
+import { generateClient } from "aws-amplify/api";
+import type { AuthUser } from "aws-amplify/auth";
+import { useRouter } from "next/navigation";
+
+import type { Schema } from "@/amplify/data/resource";
+import FormFields from "@/components/LoginForm/FormFields";
+import FormFieldsHeader from "@/components/LoginForm/FormFieldsHeader";
+import { Flex, Input, Label, SelectField } from "@aws-amplify/ui-react";
+import { useMutation, useQuery } from "@tanstack/react-query";
+
+const client = generateClient<Schema>();
+export default function PersonalFormFields({
+  user,
+}: {
+  user: AuthUser | undefined;
+}) {
+  const router = useRouter();
+  const userMutation = useMutation({
+    mutationFn: async (input: Schema["User"]["type"]) => {
+      (await client.models.User.create(input)).data;
+    },
+    onSuccess: () => {
+      // TODO: ADD TOAST
+      router.push("/");
+    },
+  });
+  const institutions = [
+    "University of Calgary",
+    "Mount Royal University",
+    "SAIT",
+    "Other",
+    "None",
+  ];
+  // Not really typesafe, but it's for a hackathon...
+  const submitForm = (formData: FormData) => {
+    const data = Object.fromEntries(formData.entries());
+    const newUserDetails = {
+      ...data,
+      id: user?.userId,
+      email: user?.signInDetails?.loginId,
+      meals: data.meals === "yes" ? true : false,
+    } as Schema["User"]["type"];
+    try {
+      userMutation.mutate(newUserDetails);
+    } catch (e) {}
+  };
```

Comment:
Let's change this to "update". As per `9575136bcbdd4a2650fba42ce677280096dce020` the user profile get's created automatically be the pre-signup lambda trigger now